target_game.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 10 21:57:08 2020
+
+@author: Aly
+"""
+# Firstly I import packages that I might need
+
+from random import randint
+
+from psychopy.visual import Window, ImageStim
+from psychopy.event import getKeys, Mouse
+from psychopy.clock import Clock
+
+# I create a function which randomly changes the position of my target
+# I chose numbers depending on my DISP size (I want the target to be always presented inside the screen)
+# then the new position is set to the target, so the target can be moved
+
+def move_target_at_random_pos(target):
+    pos_x = randint(-700, 700)
+    pos_y = randint(-400, 400)
+    
+    new_pos = (pos_x, pos_y)
+    target.setPos(new_pos)
+    
+# I call out the objects I will use and define my DISP and BGC:
+DISPSIZE = (1400,800)
+BGC = (-1,-1,-1)
+score = 0
+
+win = Window(size=DISPSIZE, units='pix', fullscr=False, color=BGC)
+target = ImageStim(win, 'target.png', size = (420, 420)) # i created this target in photoshop
+mouse = Mouse(win)
+
+mouse_click_clock = Clock()
+
+# The game starts now by drawing the target at a random location:
+
+move_target_at_random_pos(target)
+
+while True:

```

Comment:
Do you already know what this line will look like in the final version? So like, when the player has 15 points or no lives left?

Reply:
Yes, my intention is for the game to finish either with the player winning the game (having 15 points) or loosing the same (having 0 lives). I still need to implement that as it will depend on 3 variables: lives, score, and level